### PR TITLE
Feature/hb 342

### PR DESCRIPTION
**Pull request by fotinik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,39 +10,65 @@ achieve a consistent way of logging application events.
...
+Note: Its a good idea to wrap this in an env var to allow developers to use traditional logs locally:
+
+```
+unless ENV['DISABLE_AMAYSIM_LOGGER'] == 'true'
+  Rails.logger = AmaysimLogger
+end
+```
+
+3. Run `bundle install`
+4. In the application log as per usual wirh Rails.logger
```
Comment (albertjou) [inline README.md]: typo

Comment (fotinik) [inline README.md]: fixed thanks :)

File: lib/amaysim_logger.rb [hunk 2]
```diff
@@ -48,35 +48,46 @@ def logger
     delegate :level, :level=, to: :logger
     delegate :formatter, :formatter=, to: :logger
     delegate :info?, :debug?, :warn?, :error?, :unknown?, to: :logger
+    delegate :<<, to: :logger
```
Comment (athurn) [inline lib/amaysim_logger.rb]: I don't think this will work. you need to handle the this as you handle the other log methods

**Comment by yunspace** [comment]
can we merge this in, tag it with 0.0.3 and test it out? No harm since we are peged to 0.0.2 in production. Reading through the code doesn't give me much idea if this will give me the log levels I'm after. If 0.0.3 doesn't work then we'll just label it `do not use` and move improve with 0.0.4.

File: lib/amaysim_logger.rb (hunk 2 above)
Comment (fotinik) [inline lib/amaysim_logger.rb]: fixed

**Pull request merged**